core/modules/field/field.api.php

```diff
@@ -1680,6 +1680,8 @@ function hook_field_attach_delete_bundle($entity_type, $bundle, $instances) {
  *   key/value pairs:
  *   - label: The human-readable name of the storage backend.
  *   - description: A short description for the storage backend.
+ *   - validate: The name of a function which returns true if the storage
```

Comment:
This seems like an API change, so can we please add an `@since` to this docblock?